common/soroban-rpc/bin/start

```diff
@@ -3,9 +3,12 @@
 set -e
 set -o pipefail
 
-while ! (curl -sf http://localhost:8001/ | jq --exit-status '.ingest_latest_ledger > 1'); do
-  echo "Waiting for horizon to be available..."
-  sleep 1
+ while ! $(curl --silent --location --request GET 'http://localhost:8000' | \
+            jq --exit-status '.core_latest_ledger > 5 and .history_latest_ledger > 5 and (.core_latest_ledger - .history_latest_ledger) < 5' 2>/dev/null | \
```

Comment:
💭 

> (.core_latest_ledger - .history_latest_ledger) < 5

This part seems to make an assumption. Technically Horizon could be up and available and just lagging behind on a slower system. Is there another way we can determine if Horizon has completed its initial ingestion without saying it has to stay caught up?

I don't think this needs to block this merging, but something for us to think about.